.github/workflows/teardown-frontend-dev.yml

```
@@ -0,0 +1,332 @@
+
+          echo "🔍 Checking for Amplify app: ${AMPLIFY_APP_NAME}"
+
+          # Check if Amplify app exists
+          APP_INFO=$(aws amplify list-apps \
+            --query "apps[?name=='${AMPLIFY_APP_NAME}'].{appId:appId,name:name,status:status}" \
+            --output json 2>/dev/null || echo "[]")
+
+          if [[ "$APP_INFO" != "[]" ]] && [[ -n "$APP_INFO" ]]; then
+            APP_ID=$(echo "$APP_INFO" | jq -r '.[0].appId' 2>/dev/null || echo "")
+            APP_STATUS=$(echo "$APP_INFO" | jq -r '.[0].status' 2>/dev/null || echo "")
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)